frontend/cypress/e2e/importAProject.cy.ts

```diff
@@ -82,10 +82,47 @@ const checkMeanTimeToRecovery = (testId: string) => {
 }
 
 describe('Import project from file', () => {
-  it('Should import a project manually', () => {
+  it('Should import a new config project manually', () => {
     homePage.navigate()
 
-    homePage.importProjectFromFile()
+    homePage.importProjectFromFile('NewConfigFileForImporting.json')
+    cy.url().should('include', '/metrics')
+    cy.contains('Pipeline Tool').should('exist')
+    cy.get('.MuiInput-input')
```

Comment:
extract #90 #91 as a readonly fields like, home.ts#4
and refactor the next e2e test, thx